RecoPPS/ProtonReconstruction/src/ProtonReconstructionAlgorithm.cc

```diff
@@ -389,7 +389,8 @@ reco::ForwardProton ProtonReconstructionAlgorithm::reconstructFromMultiRP(const
   // save reco candidate
   const double sign_z = (armId == 0) ? +1. : -1.;  // CMS convention
   const FP::Point vertex(0., vtx_y, 0.);
-  const double cos_th = sqrt(1. - th_x * th_x - th_y * th_y);
+  const double cos_th_sq = sqrt(1. - th_x * th_x - th_y * th_y);
```

Comment:
remove `sqrt`

Reply:
Done in 3ab6103. Many thanks for catching it!